tests/client/rpcSessions.test.ts

```diff
@@ -101,81 +85,27 @@ describe('Sessions client service', () => {
     const sessionToken = await polykeyAgent.sessionManager.createToken();
     callCredentials = testUtils.createCallCredentials(sessionToken);
   });
-
   test('can request a session', async () => {
-    const requestJWT = grpcUtils.promisifyUnaryCall<sessionsPB.Token>(
+    const requestJWT = grpcUtils.promisifyUnaryCall<utilsPB.EmptyMessage>(
       client,
       client.sessionsUnlock,
     );
 
-    const passwordMessage = new sessionsPB.Password();
-    passwordMessage.setPassword(passwordFile);
-
-    const res = await requestJWT(passwordMessage);
-    expect(typeof res.getToken()).toBe('string');
-    const result = await polykeyAgent.sessionManager.verifyToken(
-      res.getToken() as SessionToken,
-    );
+    const pCall = requestJWT(new utilsPB.EmptyMessage(), callCredentials);
+    const meta = await pCall.meta;
+    const token = clientUtils.decodeAuthToSession(meta);
+    const result = await polykeyAgent.sessionManager.verifyToken(token!);
     expect(result).toBeTruthy();
   });
-  test('can refresh session', async () => {
-    const requestJWT = grpcUtils.promisifyUnaryCall<sessionsPB.Token>(
-      client,
-      client.sessionsUnlock,
-    );
-
-    const passwordMessage = new sessionsPB.Password();
-    passwordMessage.setPassword(passwordFile);
-
-    const res1 = await requestJWT(passwordMessage);
-    const token1 = res1.getToken() as SessionToken;
-    const callCredentialsRefresh = testUtils.createCallCredentials(token1);
-
-    const sessionRefresh = grpcUtils.promisifyUnaryCall<sessionsPB.Token>(
-      client,
-      client.sessionsRefresh,
-    );
-
-    await sleep(1100);
-    const emptyMessage = new utilsPB.EmptyMessage();
-    const res2 = await sessionRefresh(emptyMessage, callCredentialsRefresh);
-    expect(typeof res2.getToken()).toBe('string');
-    const token2 = res2.getToken() as SessionToken;
-    const result = await polykeyAgent.sessionManager.verifyToken(token2);
-    expect(result).toBeTruthy();
-    expect(token1).not.toEqual(token2);
-  });
-  test('actions over GRPC refresh the session', async () => {
-    // Since we refresh the token when metadata is sent, check that
-    // metadata is sent and that we can do something when it happens
-    const agentStatus = grpcUtils.promisifyUnaryCall<utilsPB.EchoMessage>(
-      client,
-      client.agentStatus,
-    );
-    const emptyMessage = new utilsPB.EmptyMessage();
-    const res = agentStatus(emptyMessage, callCredentials);
-    let check = 0;
-    res.call.on('metadata', () => (check = 1));
-    await res;
-    expect(check).toEqual(1);
-  });
-  test('session can lock all', async () => {
-    const agentStatus = grpcUtils.promisifyUnaryCall<utilsPB.EchoMessage>(
-      client,
-      client.agentStatus,
-    );
-
-    // Locking the session.
-    const sessionLockAll = grpcUtils.promisifyUnaryCall<utilsPB.EchoMessage>(
+  test('can lock all sessions', async () => {
+    const requestJWT = grpcUtils.promisifyUnaryCall<utilsPB.EmptyMessage>(
```

Comment:
Didn't we stop using the `JWT` terminology when dealing with sessions?

Reply:
Yeah that sounds familiar, I just left it as it was but I'll give it a better name now